cnote.java

```diff
@@ -0,0 +1,40 @@
+import java.util.*;
+class cnote
+{public static void main (String[] args) throws java.lang.Exception{
```

Comment:
Reformat this file with 4 space indentation and move it to the correct directory.